.github/workflows/ci.yml

```diff
@@ -32,14 +32,14 @@ jobs:
         docker tag ${{ secrets.IMAGE_NAME }}:latest ${{ secrets.IMAGE_NAME }}:${GITHUB_SHA::6}
         docker push ${{ secrets.IMAGE_NAME }}:latest
         docker push ${{ secrets.IMAGE_NAME }}:${GITHUB_SHA::6}
-        cd ../smi-conformance && docker build --no-cache -t smi-conformance:latest .
-        docker tag smi-conformance:latest smi-conformance:${GITHUB_SHA::6}
-        docker push smi-conformance:latest
-        docker push smi-conformance:${GITHUB_SHA::6}
+        cd ../smi-conformance && docker build --no-cache -t ${{ secrets.IMAGE_NAME }}:smi .
+        docker tag ${{ secrets.IMAGE_NAME }}:smi ${{ secrets.IMAGE_NAME }}:smi-${GITHUB_SHA::6}
+        docker push ${{ secrets.IMAGE_NAME }}:smi
```

Comment:
@kumarabd given the update to the tag name of `smi`, we may have to update adapter logic accordingly.